server/portal/apps/workspace/api/views.py

```diff
@@ -557,3 +457,20 @@ def get(self, request):
         )
 
         return JsonResponse({"tabs": tabs, "definitions": definitions})
+
+
+@method_decorator(login_required, name='dispatch')
+class TapisAppsView(BaseApiView):
+    def get(self, request, operation=None):
+        try:
+            client = request.user.tapis_oauth.client
+        except AttributeError:
+            return JsonResponse(
```

Comment:
is this except block needed cause the `login_required` not working right here?

Reply:
Good point, `login_required` should do the job.

Though fyi this whole view is actually not in use at the moment, but haven't removed it yet in case we do use it